Cargo.toml

```diff
@@ -66,7 +66,7 @@ flate2 = "^1.0"
 lazy_static = "^1.3"
 
 [profile.release]
-lto = true
+lto = "thin"
```

Comment:
why this change ?

Reply:
Marco asked for a patch about it, it ended up in this set, I can make a separate PR for it.